docs/external_system.md

```diff
@@ -135,65 +138,86 @@ use externally-accessible API endpoints exposed by CSM.
    variable, then `source ${SAT_VENV_PATH}/bin/activate` will need to be run before
    running any SAT commands.
 
-   ```screen
-   (venv) $ deactivate
-   $ echo export PATH=\"${SAT_VENV_PATH}/bin:${PATH}\" >> ~/.bash_profile
-   $ source ~/.bash_profile
+   ```bash
+   deactivate
+   echo export PATH=\"${SAT_VENV_PATH}/bin:${PATH}\" >> ~/.bash_profile
+   source ~/.bash_profile
    ```
 
-1. Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config`
+1. (`user@hostname>`) Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config`
    on the external system.
 
    Note that this file contains credentials to authenticate against the Kubernetes
    API as the administrative user, so it should be treated as sensitive.
 
-   ```screen
-   $ mkdir -p ~/.kube
-   $ scp ncn-m001:/etc/kubernetes/admin.conf ~/.kube/config
+   ```bash
+   mkdir -p ~/.kube
+   scp ncn-m001:/etc/kubernetes/admin.conf ~/.kube/config\
+   ```
+
+   Example output:
+
+   ```bash
    admin.conf                                       100% 5566   3.0MB/s   00:00
    ```
 
-1. Add a new entry for the hostname `kubernetes` to the external system's
-   `/etc/hosts` file.
+1. (`user@hostname>`) Find the CAN IP address on `ncn-m001` to determine the
+   corresponding `kubernetes` hostname.
+
+   - On CSM 1.2 and newer, query the IP address of the `bond0.cmn0`
+     interface.
+
+     ```bash
+     ssh ncn-m001 ip addr show bond0.cmn0
+     ```
+
+     Example output:
+
+     ```bash
+     13: bond0.cmn0@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+     link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+     inet 10.102.1.11/24 brd 10.102.1.255 scope global vlan007
+        valid_lft forever preferred_lft forever
+     inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+        valid_lft forever preferred_lft forever
+     ```
+
+   - On CSM versions prior to 1.2, query the IP address of the `vlan007` interface.
+
+     ```bash
+     ssh ncn-m001 ip addr show vlan007
+     ```
+
+     Example output:
+
+     ```bash
+     13: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+     link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+     inet 10.102.1.10/24 brd 10.102.1.255 scope global vlan007
+        valid_lft forever preferred_lft forever
+     inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+        valid_lft forever preferred_lft forever
+     ```
+
+1. (`user@hostname>`) Add a new entry for the hostname `kubernetes` to the
+   external system's `/etc/hosts` file.
```

Comment:
This step should just say something like "Set the `IP_ADDRESS` variable to the value determined in the previous step"

Reply:
Updated in next commit.